src/widgets/call-panel.jsx

```diff
@@ -155,6 +175,12 @@ class CallPanel extends React.PureComponent {
     // Start local video.
     navigator.mediaDevices.getUserMedia(this.localStreamConstraints)
       .then(stream => {
+        if (!this.localStreamConstraints.video) {
+          // Starting an audio-only call. Create a dummy video track
+          // (so video can be enabled during the call if the user desires).
+          const dummy = this.emptyVideoTrack();
+          stream.addTrack(dummy);
```

Comment:
Maybe just `stream.addTrack(this.emptyVideoTrack());` ?

Reply:
Done.